cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -490,11 +492,22 @@ type clientFactory struct {
 }
 
 func (f *clientFactory) New() stackdriver.StorageClient {
+	rb, rbcleanup := manual.GenerateAndRegisterManualResolver()
```

Comment:
The manual resolver should only be used when the user has selected it via command-line flag.